src/Microsoft.AspNet.Authentication.OpenIdConnect/OpenIdConnectHandler.cs

```diff
@@ -549,6 +561,25 @@ public OpenIdConnectHandler(HttpClient backchannel)
                     return null;
                 }
                 ticket = authorizationCodeReceivedContext.AuthenticationTicket;
+
```

Comment:
Okay. Note that this change will be useless whence you implement the hybrid flow, since you'll have to use the token response and not the authorization response at L. 569.